frappe/utils/pdf.py

```diff
@@ -14,6 +14,11 @@ def get_pdf(html, options=None, output = None):
 	html, options = prepare_options(html, options)
 	fname = os.path.join("/tmp", "frappe-pdf-{0}.pdf".format(frappe.generate_hash()))
 
+	options.update({
+		"disable-javascript": "",
```

Comment:
To make it look more like we are passing flag (which generally deals with boolean), can we set values as `True` instead of an empty string...

Reply:
I tried that first buy it doesn't work. It results in a command like  ```... --disable-javascript True ...``` instead of ```... --disable-javascript ...```
and throws error.